webapp/loggedin/modules/home/collectImportReport/nodeDefReportItem.js

```diff
@@ -69,4 +70,14 @@ const NodeDefReportItem = props => {
   )
 }
 
+const _getNodeDefPath = (survey, nodeDef, lang) => {
+  const nodeDefPathParts = []
+  Survey.visitAncestorsAndSelf(
+    nodeDef,
+    def => nodeDefPathParts.unshift(NodeDef.getLabel(def, lang) || NodeDef.getName(def))
```

Comment:
```NodeDef.getLabel``` already returns the name when the label hasn't been specified

Reply:
it's true...